openapi/files.com/openapi.yaml

```diff
@@ -1,6 +1,7 @@
 # There is a problem in responses of type double. From the API responses it sometimes maps to type string. 
 openapi: 3.0.1
 info:
+@display {label: "Files Com", iconPath: "resources/files.com.svg"} 
```

Comment:
@kapilraajP Something went wrong here. Please check.

Reply:
I corrected that with correct x-ballerina-display tag.